src/EventListener/Navigation/FaqNavigationListener.php

```diff
@@ -26,7 +26,7 @@ protected function getUrlKey()
      */
     protected function findCurrent()
     {
-        $alias = (string) Input::getAutoItem($this->getUrlKey(), false, true);
+        $alias = (string) Input::get($this->getUrlKey(), false, true);
```

Comment:
These changes are wrong, if you check what the original method did. I'm unsure it is still possible (or whether we still want to support) to disable auto items, but this would need to be `Input::get('auto_item')` if auto items are enabled. And since we still support Contao 4 technically the option still exists 🤷